magefiles/magefile.go

```diff
@@ -222,10 +221,18 @@ func (ci CI) TestE2E() error {
 		return fmt.Errorf("error when bootstrapping cluster: %v", err)
 	}
 
+	if err := registerPacServer(); err != nil {
+		return fmt.Errorf("error when registering PAC server: %v", err)
+	}
+
 	if err := RunE2ETests(); err != nil {
 		testFailure = true
 	}
 
+	if err := unregisterPacServer(); err != nil {
+		return fmt.Errorf("error when unregistering PAC server: %v", err)
```

Comment:
Due to recent issues with the sprayproxy host I have doubts about its reliability. So I think rather than returning an error here (and failing the whole CI job) I would rather log the error to the output (`klog.Errorf()`)

Reply:
I think we should still rather log the error than return it and fail the whole test.

Reply:
agree, [registerPacServer](https://github.com/redhat-appstudio/e2e-tests/blob/2cf6e8bc778958e03b10c49568259a5990340091/magefiles/magefile.go#L224) as well